src/sdk_codes.ts

```diff
@@ -0,0 +1,171 @@
+import { Logger } from './log';
+import { Target } from './types';
+
+interface SDKCodeMessages {
+  [key: number]: string;
+}
+
+const sdkCodes: SDKCodeMessages = {
```

Comment:
Probably no need for this interface since it can be replaced with `Record`
```suggestion
const sdkCodes: Record<number, string> = {
```

Reply:
You could also consider just making this into an enum

Reply:
Awesome!